packages/request-client.js/src/http-request-network.ts

```diff
@@ -24,7 +25,8 @@ export default class HttpRequestNetwork extends RequestNetwork {
    * @param options.useMockStorage When true, will use a mock storage in memory. Meant to simplify local development and should never be used in production.
    * @param options.signatureProvider Module to handle the signature. If not given it will be impossible to create new transaction (it requires to sign).
    * @param options.currencies custom currency list
-   * @param options.currencyManager custom currency manager (will override `currencies`)
+   * @param options.currencyManager custom currency manager (will override `currencies`)http
+   * @param options.skipCreateConfirmation allows to create a transaction without persisting it.
```

Comment:
Please be super precise: do we skip the confirmation of the persistence or the persistence? Your comment and naming seem to contradict each other.

Reply:
Changed to `skipPersistence` :white_check_mark: